test/support/assert_api_response.rb

```diff
@@ -30,6 +30,53 @@ def assert_api_request(method, path, params = {})
 
     assert_mock http
   end
+
+  # Asserts that the given resource can use auto-pagination.
+  #
+  # path     - API path (without `/api/v1` prefix, eg: "/messages")
+  # resource - ReamazeAPI resource name (eg: `:messages`)
+  #
+  # Example
+  #
+  #   assert_pagination "/conversations", :conversations
+  def assert_pagination(path, resource)
```

Comment:
can you make this clearer? specifically, I'm unclear why/how the expected is actually expected - how is it setup that we would expect it to meet that? I might be missing something, but i cant discern.